### ISSUE TITLE
Problem with Arduino IDE v1.8.13 (Windows 10)

### ISSUE THREAD

**Issue by venice1200** [MrBlinky/Arduboy-homemade-package#6]
Hi,   
I am using the Windows ZIP Versions of the Arduino IDE and tried the actual v1.8.13 
using the configuration from the previous (working) Versions 1.8.12 and 1.8.10.   
If I start v1.8.13 with an previously used "normal" AVR Board and choose now one of the Arduboy Boards   
I get some Java exeptions shown and the IDE 1.8.13 is no longer able to start if stopped.   
v1.8.12 is still working.

I use Package Version 1.2.7.

Is the problem related to the Arduino IDE 1.8.13, to my Windows or to the Arduboy-homemade-package?

Thanks for your Package, the Bootloader and all the other cool Arduboy stuff.

Cheers
venice1200

[Arduboy-homemade-package_Arduino_IDE_1.8.13_exeptions.txt](https://github.com/MrBlinky/Arduboy-homemade-package/files/4827812/Arduboy-homemade-package_Arduino_IDE_1.8.13_exeptions.txt)

**Comment by MrBlinky** [comment]
Thanks for the report. I just downloaded the new 1.8.13 IDE verison and can confirm the issue. I will look into it.

**Comment by MrBlinky** [comment]
Looks like the new feature of only showing compatible programmers crashed on a null pointer because there is no list of compatible programmers. Need to determine what determines the compatible programmers

**Comment by MrBlinky** [comment]
found the problem. Arduino IDE 1.8.3 doesn't like it when there is no default .build.core= line for a board in boards.txt. 

adding the following lines to the board.txt file will fix the problem.

```
arduboy-homemade.build.core=arduino:arduino
arduboy.build.core=arduino:arduino
arduboy-devkit.build.core=arduino:arduino

```
Will add a fix tomorrow

(Edit replaced . with : )

**Comment by venice1200** [comment]
Hi,
I can confirm the fix with the additional lines.
Many Thanks for your help.

**Comment by venice1200** [comment]
Just an request from my side;

I think it makes sense to rename in boards.txt the line
arduboy-homemade.name=Homemade Arduboy
in 
arduboy-homemade.name=Arduboy Homemade
just from an cosmetic point of view.

If you use the cli version of arduino the arduboy version/boards are all listed together after this change 
as the list is alphabetical.

Before
![Homemade_Arduboy](https://user-images.githubusercontent.com/43762187/85832846-19d8c900-b791-11ea-8464-c9948885e456.png)

After
![Arduobo_Homemade](https://user-images.githubusercontent.com/43762187/85832675-cebeb600-b790-11ea-8852-7f54db0b6b86.png)

**Comment by MrBlinky** [comment]
Thanks for your suggestion. But i'll keep it Homemade Arduboy. You're free to rename it in your version ofcourse.

**Comment by MrBlinky** [comment]
The issue is fixed now in 1.2.8 package
